CHANGELOG.md

```diff
@@ -6,6 +6,18 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
+- Add `rest_client_options` option to `Conjur.configuration`
+
+### Changed
+- Replaces monkey patching RestClient::Request, to always set :ssl_cert_store,
```

Comment:
Can we shorten this CHANGELOG entry a bit? Something like "Remove RestClient::Request monkeypatch to allow configuration of :ssl_cert_store to be scoped to Conjur".